database/seeder.js

```diff
@@ -0,0 +1,112 @@
+const mysql = require('mysql');
+const faker = require('faker');
+console.log(faker.finance.amount(1, 5));
+const config = require('./config.js');
+const db = mysql.createConnection(config);
+const MAX_Record = 100;
+
+const randomNumberGenerator = (max) => {
+  return Math.floor(Math.random() * Math.floor(max));
+}
+
+const tableGenerator = (table, generator, recordNum) => {
+  for (let index = 0; index < recordNum; index++) {
+    table.push(generator());
+  }
+  return table;
+}
+
+const similarListingGenerator = () => {
+  return [
+    faker.image.avatar(),
+    faker.lorem.sentence(),
+    faker.lorem.sentence(3),
+    faker.finance.amount(50, 500),
+    faker.random.number(1,5),
+
+  ]
+}
+
+const thingsToDoGenerator = () => {
+  return [
+    faker.image.avatar(),
+    faker.lorem.sentence(),
+    faker.lorem.sentence(3),
+    faker.finance.amount(50, 500),
+    faker.random.number(1,5),
+  
+  ]
+}
+
+// const offeringGenerator = () => {
```

Comment:
Not sure what the commented out sections are in this file. Are they supposed to be here?

Reply:
Eric,

My original plan was to create an offering table that would be connected to **thingsToDo** and **similarListing** tables. But, latter I ran into syntax issues at the eleventh hour before yesterday's deadline. At this MVP stage, I am trying to keep things simple. 

Thanks,
Rajat